main01.js

```diff
@@ -0,0 +1,39 @@
+let datum = document.querySelector(".datum");
+let searchbar = document.getElementById("searchbar");
+let dataStore = [];
+
+getdata();
```

Comment:
Wrap this function with this please:

document.addEventListener('DOMContentLoaded',() => {
    // handle DOMContentLoaded event
});